### ISSUE TITLE
error during export: duplicate field 'note'

### ISSUE THREAD

**Issue by jendrikw** [retorquere/zotero-better-bibtex#1636]
*Please verify **all** of the below before submitting*:

1. you have picked the right category for the issue in the previous screen.
2. You are on the latest release of Zotero
3. in the Zotero addons screen you can see that you have the latest release of BBT (https://github.com/retorquere/zotero-better-bibtex/releases/latest)
4. you are posting a single bug or feature request.
5. the issue has a subject that succinctly describes the problem or question.
6. you are available for follow-up questions and testing.
7. you have included an error-report ID here generated by reproducing the problem, selecting the problematic reference(s), right-clicking, and submitting an BBT error report from that popup menu, **not** from the `Help` menu.
8. you have included the *actual output* you got from exporting the items you sent in the debug log, not simplified fictional output or a description of the output, and a sample (*not* a description) of how you want it to look instead, again based on the actual items you selected in the previous step.

Picking the right issue category is really important. Each category (`Export`, `General error`, `Import`, `Key generation`, `Question`) has different instructions for gathering the data necessary required to resolve the issue you are experiencing

The error report is important; it gives me your current BBT settings and a copy of the problematic reference as a test case so I can best replicate your problem. Without it, I'm effectively blind.

**Report ID:** XW4VYZ4Z-euc

**Exporter used:** Better BibLaTeX

**Expected behavior:** Should export without error, as it worked around a week ago.

**Actual behavior:** I get an error during export:
```
Error: duplicate field 'note' for LichessTeamLichessEndYear: old: {Siehe auch \url{https://lichess.org/analysis}}, new: <p>https://lichess.org/analysis</p> add@chrome://zotero/content/xpcom/translation/translate_firefox.js line 425 > eval:12637:23 complete@chrome://zotero/content/xpcom/translation/translate_firefox.js line 425 > eval:12903:17 doExport@chrome://zotero/content/xpcom/translation/translate_firefox.js line 425 > eval:10797:9 rest@chrome://zotero/content/xpcom/translation/translate.js:1353:28 Zotero.Translate.Base.prototype._translateTranslatorLoaded</<@chrome://zotero/content/xpcom/translation/translate.js:1343:37 tryCatcher@resource://zotero/loader.jsm -> resource://zotero/bluebird/util.js:16:16 module.exports/Promise.prototype._settlePromiseFromHandler@resource://zotero/loader.jsm -> resource://zotero/bluebird/promise.js:512:13 module.exports/Promise.prototype._settlePromise@resource://zotero/loader.jsm -> resource://zotero/bluebird/promise.js:569:13 module.exports/Promise.prototype._settlePromise0@resource://zotero/loader.jsm -> resource://zotero/bluebird/promise.js:614:5 module.exports/Promise.prototype._settlePromises@resource://zotero/loader.jsm -> resource://zotero/bluebird/promise.js:693:13 Async.prototype._drainQueue@resource://zotero/loader.jsm -> resource://zotero/bluebird/async.js:133:13 Async.prototype._drainQueues@resource://zotero/loader.jsm -> resource://zotero/bluebird/async.js:143:5 Async/this.drainQueues@resource://zotero/loader.jsm -> resource://zotero/bluebird/async.js:17:9 From previous event: captureStackTrace@resource://zotero/loader.jsm -> resource://zotero/bluebird/debuggability.js:829:23 CapturedTrace@resource://zotero/loader.jsm -> resource://zotero/bluebird/debuggability.js:721:5 longStackTracesCaptureStackTrace@resource://zotero/loader.jsm -> resource://zotero/bluebird/debuggability.js:400:19 module.exports/Promise.method/<@resource://zotero/loader.jsm -> resource://zotero/bluebird/method.js:13:9 Zotero.Translate.Export.prototype.translate@chrome://zotero/content/xpcom/translation/translate.js:2548:12 Zotero_Translate_Export_prototype_translate@chrome://zotero-better-bibtex/content/common.js:57946:12 exportItems@chrome://zotero-better-bibtex/content/common.js:66073:9
```

This is the item which generates the error:

![image](https://user-images.githubusercontent.com/12938023/92373337-78f67a80-f0fe-11ea-81a5-fcbfd643771e.png)

**Comment by jendrikw** [comment]
I realized I had a note in Zotero itself. Deleting that fixed the issue

![image](https://user-images.githubusercontent.com/12938023/92373774-21a4da00-f0ff-11ea-90cc-81fa6bbcf57d.png)

**Comment by retorquere** [comment]
Still should not have errored out. I'm looking into it.

**Comment by retorquere** [comment]
The debug log gives me a full copy of the item I can import, BTW, and I usually add these as test cases to prevent regressions.

**Comment by retorquere** [comment]
You have a postscript that says `<not set>`; it doesn't give an error in production as postscript errors are ignored, but BBT tries to interpret that text as javascript. In your case, leaving it empty would be better.

You also have background exports disabled BTW.

**Comment by blip-bloop** [comment]
:robot: this is your friendly neighborhood build bot announcing [test build 5.2.66.7103](https://github.com/retorquere/zotero-better-bibtex/releases/download/builds/zotero-better-bibtex-5.2.66.7103.xpi) ("fixes #1636")

Install in Zotero by downloading [test build 5.2.66.7103](https://github.com/retorquere/zotero-better-bibtex/releases/download/builds/zotero-better-bibtex-5.2.66.7103.xpi), opening the Zotero "Tools" menu, selecting "Add-ons", open the gear menu in the top right, and select "Install Add-on From File...".

**Comment by retorquere** [comment]
7103 should work even if you have a note.
